### PR TITLE
Add Sushiswap to dex.trades on Spellbook

### PR DESCRIPTION
**Pull request by augustog**
Brief comments on the purpose of your changes:
- Add support for sushiswap on dex.trades on Arrakis

Tasks:
* [x] Port Sushiswap model from Caladan to Arrakis
* [x] Integrate into dex.trades
* [x] Update metadata
* [ ] Write tests

*For Dune Engine V2*
I've checked that:
General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
* [x] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [x] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [x] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/sushiswap/ethereum/sushiswap_ethereum_schema.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      - &token_sold_amount_raw
+        name: token_sold_amount_raw
+        description: "Raw value of the token sold at time of execution in the original currency"
+      - &amount_usd
+        name: amount_usd
+        description: "USD value of the trade at time of execution"
+      - &token_bought_address
+        name: token_bought_address
+        description: "Contract address of the token bought"
+        tests:
```
Comment (jeff-dude) [inline models/sushiswap/ethereum/sushiswap_ethereum_schema.yml:57]: in order for this test to run, you'll need to add sushiswap tx's to our dex trades seed file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv

there are uniswap examples in there. typically on the app UI, you can find an analytics page, maybe this one:
https://analytics.sushi.com/
which links to tx's ideally, where you can grab info and apply to seed file

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Comment by jeff-dude** [comment]
there are also some dupes based on logic for unique identifiers -- may be worth a look into source data:
```
select
    unique_trade_id as unique_field,
    count(*) as n_records

from dbt_jeff_sushiswap_ethereum.trades
where unique_trade_id is not null
group by unique_trade_id
having count(*) > 1
```

**Comment by jeff-dude** [comment]
@augustog are you comfortable continuing this PR? it'd be helpful to merge main into your branch, get latest changes from spellbook included, then start to look into pieces of feedback.

happy to help, if needed, just let me know 👍

File: models/sushiswap/ethereum/sushiswap_ethereum_schema.yml (hunk 1 above)
Comment (jeff-dude) [reply models/sushiswap/ethereum/sushiswap_ethereum_schema.yml:57]: that's a tough one to find in etherscan (at least for me too). i believe it's under the ```logs``` tab -- you can use v1 engine to query that ```tx_hash```, find ```evt_index``` values and try to find on that tab in etherscan

**Comment by augustog** [comment]
mistakes were made with version management (I am evidently rusty with git)
will clean up and create a new PR

**Pull request closed without merging**